lib/matplotlib/backends/_backend_pdf_ps.py

```diff
@@ -16,6 +19,28 @@ def _cached_get_afm_from_fname(fname):
         return AFM(fh)
 
 
+def getSubset(fontfile, characters):
```

Comment:
This is not a valid PEP8 name. Also, it's not a descriptive name; `matplotlib.backends._backend_pdf_ps.getSubset` does not really indicate what you're subsetting.

Reply:
I changed the name to `get_glyphs_subset`